lib/spack/spack/test/modules/tcl.py

```diff
@@ -108,6 +108,18 @@ def test_alter_environment(self, modulefile_content, module_configuration, confi
         assert len([x for x in content if "module load foo/bar" in x]) == 1
         assert len([x for x in content if "setenv LIBDWARF_ROOT" in x]) == 1
 
+    def test_prepend_path_separator(self, modulefile_content, module_configuration):
+        """Tests modifications to run-time environment."""
+
+        module_configuration("module_path_separator")
+        content = modulefile_content("module-path-separator")
+
+        for line in content:
+            if re.match(r"[a-z]+-path .* COLON ", line):
+                assert ' --delim ":" ' in line
+            elif re.match(r"[a-z]+-path .* SEMICOLON ", line):
+                assert ' --delim ";" ' in line
```

Comment:
Code looks good to me, but I think an empty module file would pass this test?

Reply:
You are right. I tried to copy/paste the same test used to for lmod, but I think it is better to change the logic and explicitly test the explicit output like on on `test_alter_environment`